pkg/channel/event_receiver.go

```diff
@@ -199,7 +199,7 @@ func (r *EventReceiver) ServeHTTP(response nethttp.ResponseWriter, request *neth
 		ReportEventCountMetricsForDispatchError(err, r.reporter, &args)
 		return
 	}
-	r.logger.Debug("Request mapped to channel", zap.String("channel", channel.String()))
+	r.logger.Info("Request mapped to channel", zap.String("channel", channel.String()))
```

Comment:
Why are we switching this from `Debug` to `Info`?

Reply:
Same as above :facepalm:. Reverted it...